devices/Mcp7940xx/Mcp79401.cs

```diff
@@ -0,0 +1,134 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using Iot.Device.Common;
+
+namespace Iot.Device.Mcp7940xx
+{
+    /// <summary>
+    /// Battery-Backed I2C Real-Time Clock/Calendar with Pre-Programmed EUI-48 MAC ID.
+    /// </summary>
+    public class Mcp79401 : Mcp7940n
+    {
+        /// <summary>
+        /// Lower address of EEPROM memory region.
+        /// </summary>
+        private const byte LowerAddressBoundEEPROM = 0x0;
+
+        /// <summary>
+        /// Upper address of EEPROM memory region.
+        /// </summary>
+        private const byte UpperAddressBoundEEPROM = 0xEF;
+
+        /// <summary>
+        /// EEPROM Address of the EUI Node Address.
+        /// </summary>
+        protected const byte EuiAddress = 0xF0;
+
+        /// <summary>
+        /// Default I2C address for EEPROM.
+        /// </summary>
+        protected const byte EepromI2cAddress = 0b1010_0111;
+
+        /// <summary>
+        /// The underlying I2C device used for accessing the EEPROM address space.
+        /// </summary>
+        protected readonly I2cDevice _EepromI2cDevice;
```

Comment:
StyleCop didn't like the first letter being lower case on a protected read only variable. This was also an issue with the protected _I2cDevice variable (instead of _i2cDevice) declared in Mcp7940m. I can switch the name to match convention but I wasn't quite sure what would be correct in this instance. Error I am getting when I compile is below...

SA1304 : CSharp.Naming : Readonly variables that are not declared private must start with an upper-case letter: _eepromI2cDevice.